public_html/wp-content/plugins/wc-fonts/wc-fonts.php

```diff
@@ -36,7 +36,8 @@ public function wp_head_typekit() {
 			return;
 		}
 
-		printf( '<script type="text/javascript" src="https://use.typekit.com/%s.js"></script>' . "\n", $this->options['typekit-id'] );
+		// phpcs:ignore WordPress.WP.EnqueuedResources.NonEnqueuedScript
+		printf( '<script type="text/javascript" src="https://use.typekit.com/%s.js"></script>' . "\n", esc_attr( $this->options['typekit-id'] ) );
```

Comment:
`sanitize_key` might be a better function here than `esc_attr`, although I don't know if phpcs would recognize it as an escaping function. Same for the Font Awesome kit output on line 66.

Reply:
ah, yeah - I need a cheatsheet of all the sanitization functions (oh hey, [there is one](https://developer.wordpress.org/themes/theme-security/data-sanitization-escaping/))